src/platform/backends/shared/linux/backend_utils.cpp

```diff
@@ -198,3 +188,30 @@ QString mp::backend::cpu_arch()
 
     return cpu_to_arch.value(QSysInfo::currentCpuArchitecture());
 }
+
+void mp::backend::check_for_kvm_support()
+{
+    QProcess check_kvm;
+    check_kvm.setProcessChannelMode(QProcess::MergedChannels);
+    check_kvm.start("check_kvm_support");
+    check_kvm.waitForFinished();
+
+    if (check_kvm.exitCode() == 1)
+    {
+        throw std::runtime_error(check_kvm.readAll().trimmed().toStdString());
+    }
+}
+
+void mp::backend::check_if_kvm_is_in_use()
+{
+    auto kvm_fd = open("/dev/kvm", O_RDWR | O_CLOEXEC);
+    auto ret = ioctl(kvm_fd, KVM_CREATE_VM, (unsigned long)0);
+
+    close(kvm_fd);
+
+    if (ret == -1 && errno == EBUSY)
+        throw std::runtime_error(
+            "Another hypervisor is currently running. Please shut it down before starting a Multipass instance.");
```

Comment:
:+1:

I would word it differently though, people shouldn't need to know what a hypervisor is to understand this message ;)

Something like… "Virtual Machine manager"?

Reply:
Yeah, I figured the wording would need some modification.  Although "Virtual Machine manager" isn't really accurate either, I guess it gets the point across to a non-technical user better than "hypervisor", so we'll go with that.

Reply:
Done